admin-dev/.htaccess

```diff
@@ -54,8 +54,8 @@ DirectoryIndex index.php
     # - disable this feature by commenting the following 2 lines or
     # - use Apache >= 2.3.9 and replace all L flags by END flags and remove the
     #   following RewriteCond (best solution)
-    RewriteCond %{ENV:REDIRECT_STATUS} ^$
-    RewriteRule ^index\.php(/(.*)) %{ENV:BASE}/$2 [R=301,L]
+    # RewriteCond %{ENV:REDIRECT_STATUS} ^$
+    # RewriteRule ^index\.php(/(.*)) %{ENV:BASE}/$2 [R=301,L]
```

Comment:
If I remember well, this won't break the Symfony routes. However, the merchant won't be redirected automatically to a "more beautiful" URL if it has index.php in it.

Currently, going to the URL **/admin-dev/index.php/modules/catalog** will redirect you directly to **/admin-dev/modules/catalog** before processing.